soql-analyzer/src/main/scala/com/socrata/soql/SoQLAnalyzer.scala

```diff
@@ -176,6 +178,7 @@ class SoQLAnalyzer[Type](typeInfo: TypeInfo[Type],
     * @throws com.socrata.soql.exceptions.SoQLException if the query is syntactically or semantically erroneous.
     * @return The analysis of the query.  Note this should be appended to `sourceFrom` to form a full query-chain. */
   def analyzeSplitQuery(distinct: Boolean,
+                        distinctOn: Option[String],
```

Comment:
Does `true, Some(...)` make sense as values here?  If not, some other way to express this would be better (ditto everywhere else distinct + distinctOn exists in parallel)

Reply:
No, they cannot be used simultaneously.

Reply:
How about distinct: Either[Boolean, Seq[Expression]]?